numpy/core/src/_simd/_simd_inc_easyintrin.h

```diff
@@ -0,0 +1,228 @@
+#if !NPY_SIMD
+    #error "Not a standalone header, only works through 'simd.dispatch.c.src'"
+#endif
+
+#define SIMD_INTRIN_DEF(NAME) \
+    { NPY_TOSTRING(NAME), simd__intrin_##NAME, METH_VARARGS, NULL } , // comma
+
+static int simd__no_arguments(PyObject *args, const char* method_name)
+{
+    if (args == NULL) {
+        return 0;
+    }
+    assert(PyTuple_Check(args));
+    Py_ssize_t obj_arg_len = PyTuple_GET_SIZE(args);
+    if (obj_arg_len != 0) {
+        PyErr_Format(PyExc_RuntimeError,
+            "%s(), takes no arguments, given(%d)", method_name, obj_arg_len
+        );
+        return -1;
+    }
+    return 0;
+}
+
+#define SIMD_IMPL_INTRIN_0(NAME, RET)                     \
+    static PyObject *simd__intrin_##NAME                  \
+    (PyObject* NPY_UNUSED(self), PyObject *args)          \
+    {                                                     \
+        if (simd__no_arguments(                           \
+            args, NPY_TOSTRING(NAME)                      \
+        )) return NULL;                                   \
```

Comment:
No real need for `simd__no_arguments` here, `PyArg_ParseTuple` can handle it
```suggestion
        if (!PyArg_ParseTuple(args, ":" NPY_TOSTRING(NAME))) return NULL;                                   \
```

Reply:
done